app/views/items/browse.html.erb

```diff
@@ -1,41 +1,22 @@
 <h2>Browse</h2>
 
-<p><%= link_to "View all items in the archive", search_path %> or use the following categories to facilitate browsing:</p>
+<p>
+  <%= link_to "All Items in the Archive", search_path, html_options = {class: "btn btn-lg btn-default"} %>
+</p>
 
-<div>
-
-  <!-- tab list -->
-  <ul class="nav nav-tabs" role="tablist">
-    <% Facets.facet_info.each_with_index do |(facet_name, facet_info), index| %>
-      <li role="presentation" <%= "class=active" if index == 0 %>>
-        <% id = datify_name facet_name %>
-        <a href="#<%= id %>" aria-controls="<%= id %>" role="tab" data-toggle="tab">
-          <%= facet_info["label"] %>
-        </a>
-      </li>
-    <% end %>
-  </ul>
-
-  <!-- content -->
-  <div class="tab-content">
-    <% Facets.facet_info.each_with_index do |(facet_name, facet_info), index| %>
+<h3>Browse by:</h3>
+<div class="row bottom-buffer-lg">
+  <% Facets.facet_info.each_with_index do |(facet_name, facet_info), index| %>
```

Comment:
A couple times now in past projects when we've needed to create rows of a certain number of items, we've used the ruby `each_slice` method to cut it into 3.  I think that could work nicely here to avoid having the modulus at the bottom with a closing / opening div, etc.

Here's an example, albeit with some funky spacing:  https://github.com/CDRH/hopewell_middleman/blob/master/source/catalog_cards.html.erb#L8